src/main/java/com/sprint/mission/discodeit/entity/base/baseUpdatableEntity.java

```diff
@@ -0,0 +1,15 @@
+package com.sprint.mission.discodeit.entity.base;
+
+import jakarta.persistence.MappedSuperclass;
+import java.time.Instant;
+import lombok.Getter;
+import org.springframework.data.annotation.LastModifiedDate;
+
+@Getter
+@MappedSuperclass
+public abstract class baseUpdatableEntity extends BaseEntity {
```

Comment:
```suggestion
public abstract class BaseUpdatableEntity extends BaseEntity {
```

클래스 이름은 보통 대문자로 시작합니다!